src/lib/viewers/controls/color-picker/ColorPickerPalette.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import noop from 'lodash/noop';
+
+import ControlsBar from '../controls-bar';
+
+import './ColorPickerPalette.scss';
+
+export type Props = {
+    onColorSelect: () => void; // TODO: this method wil need to take in a color string param
+};
+
+export default function ColorPickerPalette({ onColorSelect = noop }: Props): JSX.Element {
+    const colors = ['#0061d5', '#26c281', '#ed3757', '#f5b31b', '#ffd700', '#4826c2'];
```

Comment:
These colors will be moved to a better location (AnnotationModule) after this change has been merged.